File: 351003/Egorov/app/models/creator.py
from dataclasses import dataclass
from enum import Enum


class CreatorRole(str, Enum):
    ADMIN = "ADMIN"
    CUSTOMER = "CUSTOMER"


@dataclass
class Creator:
    id: int | None = None
    login: str = ""
    password: str = ""
    first_name: str = ""
    last_name: str = ""
    role: CreatorRole = CreatorRole.CUSTOMER
    name: str = ""
    email: str = ""

File: 351004/Leshok/Lab6/models/Creator.js
// models/Creator.js
class Creator {
  constructor(id, login, password, firstName, lastName, role) {
    this.id = id;
    this.login = login;
    this.password = password;
    this.firstName = firstName;
    this.lastName = lastName;
    this.role = role; // 'ADMIN' или 'CUSTOMER'
    this.createdAt = new Date().toISOString();
    this.updatedAt = new Date().toISOString();
  }
}
module.exports = Creator;